BinaryEncoding.md

```diff
@@ -335,9 +332,9 @@ The `br_table` operator has an immediate operand which is encoded as follows:
 
 | Field | Type | Description |
 | ---- | ---- | ---- |
-| target_count | `uint16` | number of targets in the target_table |
-| target_table | `uint16*` | target entries that indicate an outer block or loop to which to break |
-| default_target | `uint16` | an outer block or loop to which to break in the default case |
+| target_count | `varuint32` | number of targets in the target_table |
+| target_table | `varuint32*` | target entries that indicate an outer block or loop to which to break |
```

Comment:
Yeah, definitely seems like we'll hit `uint16_t` limit as time goes to infinity.  I think there's a famous example of this from Microsoft lore back in the day.  The binary format is already not directly interpretable with any efficiency and wants a translation to bytecode which could normalize.  Within a compiler, decoding a sequence of LEB128 is no big deal.